src/transformers/modeling_utils.py

```diff
@@ -1560,19 +1560,23 @@ def _autoset_attn_implementation(
     ):
         """
         Automatically checks and dispatches to a default attention implementation. In order of priority:
-            1. An implementation specified in `config._attn_implementation` (due for example to the argument attn_implementation="sdpa" in from_pretrained).
-            2. DEPRECATED: if use_flash_attention_2 is set to `True` and `flash_attn` is available, flash attention. (`LlamaFlashAttention` for example)
+            1. An implementation specified in `config._attn_implementation` (due for example to the argument
```

Comment:
All changes in this file are line breaks to keep <120 chars/line, unless specified